e2e_tests/common.py

```diff
@@ -30,7 +30,8 @@ def get_arch_messages(response_json):
     arch_messages = []
     if response_json and "metadata" in response_json:
         # load arch_state from metadata
-        arch_state_str = response_json.get("metadata", {}).get(ARCH_STATE_HEADER, "{}")
+        arch_state_str = response_json.get("metadata") or {}
```

Comment:
this would do the same 

`response_json.get("metadata", {})`

Reply:
I got a weird bug when response_json['metadata'] = Null